src/main/java/ch/uzh/ifi/hase/soprafs22/controller/InGameWebsocketController.java

```diff
@@ -28,21 +29,18 @@ public class InGameWebsocketController {
     }
 
 
-    // TODO: need to add {uuid} and notify all other players (refer to test function below)
-    @MessageMapping("/websocket/move")
-    @SendTo("/client/move")
-    public MoveGetDTO move(MovePostDTO MovePostDTO, Principal principal) throws Exception {
-        // get move from the client
+    @MessageMapping("/websocket/{uuid}/move")
+    public void move(@DestinationVariable String uuid, MovePostDTO MovePostDTO, Principal principal) throws Exception {
+        // Get move, username, game
         Move move = DTOMapper.INSTANCE.convertMovePostDTOtoEntity(MovePostDTO);
-
-        // verify move validity and add Player details
         String username = principal.getName();
-        move = inGameWebsocketService.verifyMove(move, username);
+        Game userGame = gameService.getGameByUuid(uuid);
```

Comment:
we will have to make sure that the user is actually in the game right? so maybe extend the function to `gameService.getGameByUuid(uuid, playerName)` ? 😄

Reply:
Yeah true, just implemented that